libsolidity/interface/OptimiserSettings.h

```diff
@@ -41,7 +41,7 @@ struct OptimiserSettings
 
 			// should have good "compilability" property here.
 
-			"eul"                      // Run functional expression inliner
+			"peul"                     // Run functional expression inliner
```

Comment:
Is the expression inliner powerful enough to complete the p step?

Reply:
Not generally. But putting this sequence is giving better results than putting it before FullInliner (`i`) Don't know why exactly.

Reply:
Ok! I guess it doesn't matter much, the full inliner will do its job eventually.